project_start.py

```diff
@@ -281,43 +321,47 @@ def load_args():
     phab = Phab(config["phab"], args.dry_run)
     with open(args.project_file[0], newline="") as file_:
         projects_reader = csv.DictReader(file_, delimiter="\t")
-        for unsanitized_project_information in projects_reader:
-            project_information = sanitize(unsanitized_project_information)
-            if project_information[project_columns["skip"]]:
-                logging.info(
-                    "Skipping '{}', marked as inactive.".format(
-                        project_information[project_columns["english_name"]]))
-                continue
-            superproject = project_information[
-                project_columns["super_project"]]
-            project_name = project_information[
-                project_columns["english_name"]]
-            if superproject:
-                # Don't add anything for subprojects.
-                continue
-            if project_name not in goals:
-                logging.warn(
-                    "Project name '{}' found in projects file, but not in goals file. It will not be created.".format(project_name)  # noqa: E501
+        if args.project:
+            # process a single project only
+            single_project = None
+            for unsanitized_project_information in projects_reader:
```

Comment:
This can be added to the existing for loop if you do something like this:
```
        for unsanitized_project_information in projects_reader:
            project_information = sanitize(unsanitized_project_information)
            if args.project:
                if args.project not in (
                    project_information[project_columns["swedish_name"]],
                    project_information[project_columns["english_name"]]
                ):
                    continue
            elif project_information[project_columns["skip"]]:
                # handle skip outside of process_project to allow
                # specifying a single project to override the skip
                # value.
                logging.info(
                    "Skipping '{}', marked as inactive.".format(
                        project_information[
                            project_columns["english_name"]]))
                continue
            process_project(project_information, project_columns)
```

Reply:
Nice. Only thing which is lost is the "warn if single project not found" and "output info about needing to manually update some pages" but that can be added to the above.